homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -119,6 +123,11 @@ def update(self):
             if equaliser >= len(temescal.equalisers):
                 temescal.equalisers.append("unknown " + str(equaliser))
 
+    @property
+    def unique_id(self):
+        """Return the device's unique ID."""
+        return "LG_soundbar_%s" % self._uuid
```

Comment:
The entity registry is aware of both the integration domain and the platform domain. We can remove `"LG_soundbar_"`.

Reply:
Normally we don't want to mix fixes and feature additions in the same PR.

Reply:
Ack.